source/helpers/on-replaced-element.ts

```diff
@@ -9,18 +9,26 @@ Tracks the replacement of an element, identified via selector.
 @param callback The function to call after it's replaced
 */
 
-export default async function onReplacedElement(selector: string, callback: VoidCallback): Promise<void> {
+export default async function onReplacedElement(
+	selector: string,
```

Comment:
Shouldn't this be changed to an object?  Isn't it a boolean trap?

Reply:
I'm not sure I follow

Reply:
https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

Reply:
Oh you meant to comment on the `callNow` line since there's no boolean here 😃 

Good point, changing

Reply:
Sorry I was on mobile and selected the wrong line.